cluster-scope/base/core/namespaces/thoth-cre-integration/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- namespace.yaml
+- ../../../../components/project-admin-rolebindings/thoth
```

Comment:
This is a `kind: Component`, not a `kind: Kustomization` -> it should be listed in the `components:` list at the end of this file, not here in `resources:`. Did automation created this?

Reply:
yes

Reply:
/cc @durandom @mimotej

Reply:
@tumido I believe there is an issue in the script here should be the fix: https://github.com/operate-first/apps/pull/2705